spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubTemplate.java

```diff
@@ -46,16 +46,21 @@
 
 	private ExecutorProvider executorProvider;
 
-	private MessageConverter converter;
+	private MessageConverter messageConverter;
 
 	private int concurrentProducers = 1;
 
 	public PubSubTemplate(GoogleCredentials credentials, String projectId) {
```

Comment:
Why has the import order changed? That's not what our checkstyle format supports. Here's our current model. I know we are missing the checkstyle file, but that's not the right import order.

java.*
<blank line>
javax.*
<blank line>
all other imports
<blank line>
org.springframework.*
<blank line>
all other static imports